python/external_task_api_client/external_task_api_client_service.py

```diff
@@ -0,0 +1,71 @@
+import aiohttp
+import asyncio
+import json
+
+
+class ExternalTaskApiClientService:
+
+    def __init__(self, baseUrl):
+        self.__baseUrl = baseUrl
+
+    async def extendLock(self, identity, workerId, externalTaskId, additionalDuration):
```

Comment:
Snake_Case in Python nutzen.

```suggestion
    async def extend_lock(self, identity, worker_id, external_task_id, additional_duration):
```